src/scripts/main.js

```diff
@@ -1,3 +1,38 @@
 'use strict';
 
-// write code here
+const tableHeader = document.querySelector('tr');
+const tableBody = document.querySelector('tbody');
+const toNumber = (string) => {
+  if (typeof string === 'string') {
+    const clearNum = string.replace(/[$, ]/g, '');
+
+    return parseInt(clearNum);
+  }
+};
+
+tableHeader.addEventListener('click', (e) => {
```

Comment:
Adding an event listener directly to the 'tableHeader' assumes that the click will always be on a 'th' element. It might be more robust to delegate the event from the 'tr' to the 'th' elements within it.